### PR TITLE
EAA|Verdictd: update readme

### PR DESCRIPTION
**Pull request by liangzhou121**
Update readme file according to verdictd's latest implementation

### CODE REVIEW

File: eaa/verdictd/README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+# Introduction
 
-Verdictd is a server(as RATS shelterd) which creates an m-TLS(Mutal Transport Layer Security) connection with Attestation Agent via remote attestation (aka "[RA-TLS](https://raw.githubusercontent.com/cloud-security-research/sgx-ra-tls/master/whitepaper.pdf)").
-Mainly functions:
-- Handle "decryption" and "get KEK" requests from Attestation Agent, and response with corresponding results.
-- Launch wrap/unwrap gRPC service which can be used by ocicrypto [containers/ocicrypto](https://github.com/containers/ocicrypt).
-
-## Design
+EAA Verdictd is a remote attestation implementation comprising of a set of building blocks that utilize Intel/AMD Security features to discover, attest, and enable critical foundation security and confidential computing use-cases. 
+It relies on [Rats-tls](https://github.com/alibaba/inclavare-containers/tree/master/rats-tls) to applies the remote attestation fundamentals and standard specifications to maintain a platform data collection service and an efficient verification engine to perform comprehensive trust evaluations.
```
Comment (haosanzi) [inline eaa/verdictd/README.md:5]: `to applies`. ---> `to apply`

File: eaa/verdictd/README.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-
-## Design
+EAA Verdictd is a remote attestation implementation comprising of a set of building blocks that utilize Intel/AMD Security features to discover, attest, and enable critical foundation security and confidential computing use-cases. 
+It relies on [Rats-tls](https://github.com/alibaba/inclavare-containers/tree/master/rats-tls) to applies the remote attestation fundamentals and standard specifications to maintain a platform data collection service and an efficient verification engine to perform comprehensive trust evaluations. 
+These trust evaluations can be used to govern different trust and security policies applied to any given workload.
 
-TODO
+Verdictd creates an m-TLS(Mutal Transport Layer Security) connection with [Attestation Agent](https://github.com/confidential-containers/attestation-agent) via remote attestation.
+Mainly functions:
+- Implemented EAA protocol which includes "decryption" and "get KEK" requests.
```
Comment (haosanzi) [inline eaa/verdictd/README.md:10]: `get` -> `gets`

File: eaa/verdictd/README.md [hunk 3]
```diff
@@ -43,37 +43,67 @@ yum install -y clang-libs clang-devel
...
+cat <<- EOF >/opt/verdictd/keys/84688df7-2c0c-40fa-956b-29d8e74d16c0
+1234567890123456789012345678901
+EOF
+
+# Launch Verdictd
+verdictd --gRPC 127.0.0.1:50001
+
+skopeo --insecure-policy copy docker://docker.io/library/alpine:latest oci:alpine
+export OCICRYPT_KEYPROVIDER_CONFIG=/etc/containerd/ocicrypt/ocicrypt_keyprovider.conf
+# generate encrypted image
```
Comment (haosanzi) [inline eaa/verdictd/README.md:107]: Please add an empty line in line 107

File: eaa/verdictd/README.md [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+Verdictd creates an m-TLS(Mutal Transport Layer Security) connection with [Attestation Agent](https://github.com/confidential-containers/attestation-agent) via remote attestation.
+Mainly functions:
+- Implemented EAA protocol which includes "decryption" and "get KEK" requests.
+- Implemented a ocicrypto [containers/ocicrypto](https://github.com/containers/ocicrypt) compatible gRPC service.
+- Implemented a grpc service which can be used to config OPA's policy files.
 
-## Installation
+# Design
 
 TODO
```
Comment (haosanzi) [inline eaa/verdictd/README.md:16]: The [pr](https://github.com/alibaba/inclavare-containers/pull/1495). has provided design doc for verdicd. Please add link .

File: eaa/verdictd/README.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
-# Verdictd
 
-## Introduction
+# Introduction
 
-Verdictd is a server(as RATS shelterd) which creates an m-TLS(Mutal Transport Layer Security) connection with Attestation Agent via remote attestation (aka "[RA-TLS](https://raw.githubusercontent.com/cloud-security-research/sgx-ra-tls/master/whitepaper.pdf)").
-Mainly functions:
```
Comment (jiazhang0) [inline eaa/verdictd/README.md:6]: Use RATS-TLS instead of Rats-tls.

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**